configs/supertokens.json

```
@@ -1,6 +1,18 @@
 {
   "index_name": "supertokens",
   "start_urls": [
+    {
+      "url": "https://supertokens.io/docs/community/introduction",
+      "tags": [
+        "community"
+      ]
+    },
+    {
+      "url": "https://supertokens.io/docs/community/",
+      "tags": [
+        "community"
+      ]
+    },
```

```suggestion
    {
      "url": "https://supertokens.io/docs/community/",
      "tags": [
        "community"
      ]
    },
```

This should be enough, since it works as a path to match, anything under the endpoint `https://supertokens.io/docs/community/` will be tagged as `community`, same goes for all the other URLs

---

When running the crawler locally we were having issues where, when just giving the https://supertokens.io/docs/community/  URL only the https://supertokens.io/docs/community/introduction would be crawled and the remaining endpoints would not

---

Adding the root of the domain also helps sometime for missing URLs in case of client-side rendering.

In that case adding `"https://supertokens.io/docs/"` to the end of the `start_urls` seems to work for me locally

---

Alright, cool will update the PR with the change